### PR TITLE
added forwards to the complexMessage Section

### PR DESCRIPTION
**Pull request by savage4618**
I added message forward information to the complexMessage function based on information from the commit: https://github.com/botlabs-gg/yagpdb/pull/1764 
**Terms**
- [x] I have read and understood this project's [Contributing Guidelines](CONTRIBUTING.md)

Signed off by: savage4618 <randomdad@randomdad.xyz>

### CODE REVIEW

File: content/docs/reference/templates/functions.md [hunk 1]
```diff
@@ -1214,6 +1214,7 @@ Creates a complex message object for use in [sendMessage](#sendmessage) function
 - `silent`: whether to suppress push and desktop notifications.
 - `menus`: a single [select menu object](#cmenu).
 - `buttons`: a slice of [button objects](#cbutton).
+- `forward`: must contain an sdict with a channel and message value
```
Comment (jo3-l) [inline content/docs/reference/templates/functions.md:1217]: "`forward`: a sdict containing `channel` and `message` keys, specifying the message to forward"?

File: content/docs/reference/templates/functions.md [hunk 2]
```diff
@@ -1241,6 +1242,17 @@ the triggering message and ping the author of that message, but suppress the res
 {{ sendMessage nil $message }}
 ```
 
+Example of a message forward:
+
+```yag
+{{ sendMessage nil (complexMessage
+    "forward" (sdict
+        "channel" .Channel.ID
+        "message" <messageID>
```
Comment (jo3-l) [inline content/docs/reference/templates/functions.md:1251]: Not sure if we want placeholders in examples--ideally they should be valid code as is. Can we use `.Message.ID` instead?

**Comment by savage4618** [comment]
suggestions implemented, also "rebased" (I didn't rebase, I just merged because I have no idea what I'm doing)

**Review by jo3-l — APPROVED** [review]
LGTM, thanks!

**Pull request merged**